src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -368,25 +374,4 @@ final class OxfordNanoporeMeasurement {
     String getRelativePath() {
         return this.measurementFolder.relativePath
     }
-
-    /*
-    Inner class that contains the logic for the metadata validation
-     */
-    private static class MetaData {
-
-        private static final SCHEMA = "/schemas/ont-metadata.schema.json"
```

Comment:
Since we defined the schema I think it would make sense to use it. How about using the JSON Slurper?

Reply:
I did not manage to validate a schema with JSON Slurper. I won't address that. Please feel free to make a suggestion

Reply:
I pushed my suggestion as discussed, this way we use the schema and do not need to hardcode the keys